contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -75,8 +75,7 @@ def __init__(self, **options):
     def _export(self, batch, event=None):  # pragma: NO COVER
         try:
             if batch:
-                envelopes = [self.log_record_to_envelope(x) for x in batch]
-                envelopes = self.apply_telemetry_processors(envelopes)
+                envelopes = self.apply_telemetry_processors(batch)
```

Comment:
This doesn't work because `apply_telemetry_processors` is expecting an `Envelope` data type.

Reply:
It does work; the batch is a list of `Envelope` data type objects as I converted them already before putting them on the queue.